lib/RpiService/GpioOutput/GpioSwitch.js

```diff
@@ -59,8 +59,22 @@ class GpioSwitch extends GpioOutput {
     }
   }
 
+  async init () {
+    if (this.pulse != null) {
+      try {
+        this.debug('GpioSwitch init gpio %d: setting %s', this.gpio, this.params.reversed ? 'high' : 'low')
+        await this.pi.command(
+          PigpioClient.commands.WRITE, this.gpio, this.params.reversed ? 1 : 0
+        )
+      } catch (error) {
+        this.warn(error)
+      }
+    }
+    return super.init()
```

Comment:
In the case the `PigpioClient.commands.WRITE` fails, in the example of a pusled reverse switch controlling a door opener, I'm thinking I really don't want the `super.init()` to run! 🤔